securedrop_client/logic.py

```diff
@@ -334,7 +337,7 @@ def on_authenticate_success(self, result):
         self.gui.show_main_window(user)
         self.update_sources()
         self.api_job_queue.login(self.api)
-        self.sync_api()
```

Comment:
this is where we used to call sync_api on login which i replaced with starting the background sync thread